src/components/IconStack/styles.styl

```
@@ -0,0 +1,10 @@
+.IconStack-wrapper {
+    position relative
```

Stylelint (17.4.0)

[error] Unknown word position (CssSyntaxError)

(CssSyntaxError)